app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/ui/AddCoInsuredBottomSheetContent.kt

```diff
@@ -226,6 +233,37 @@ private fun FetchFromSsnFields(
   }
 }
 
+private fun personNummerInputFormatter(text: AnnotatedString, mask: String): TransformedText {
+  val trimmed = if (text.text.length >= 12) text.text.substring(0..11) else text.text
+
+  val annotatedString = AnnotatedString.Builder().run {
+    for (i in trimmed.indices) {
+      append(trimmed[i])
+      if (i == 7) {
+        append("-")
+      }
+    }
+    pushStyle(SpanStyle(color = Color.LightGray))
```

Comment:
Is there some `withStyle` dsl function you can use here to denote that only that one part wants this color?

Also, we probably want to use SurfaceVariant from our color scheme here to make sure it both looks the same as other parts of the app, but also dark and light mode look ok too.

Reply:
Yeah, my point was mostly that if we then add another thing below it would also get this style, but it may not be what we actually want to happen. With the `withStyle` and the block as you did it now it is more clear which parts need to have that style, so adding something else would be a clearer decision of it it goes inside or outside of that lambda